scss/helpers/_icon-link.scss

```diff
@@ -0,0 +1,24 @@
+.icon-link {
+  display: inline-flex;
+  gap: .375rem;
```

Comment:
TODO: should be the equivalent of 5px instead of 6

Reply:
Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1854/commits/03ea9f696e6a84bd01fb59689c56f5c99bb7c5b6